lnrpc/lightning.proto

```diff
@@ -3381,6 +3381,10 @@ message AMP {
 }
 
 message AddInvoiceResponse {
+    /*
+    The hash of the preimage. When using REST, this field must be encoded
```

Comment:
Do we need this, since it's only the _response_ message? The user will never specify this field, only get it as a return value.

Reply:
True. I will remove the comment.